lib/livebook_web/live/session_live.ex

```diff
@@ -499,6 +499,14 @@ defmodule LivebookWeb.SessionLive do
     {:noreply, socket}
   end
 
+  def handle_event("set_cell_value", %{"cell_id" => cell_id} = value, socket)
+  when not is_map_key(value, "value") do
+    # Handle unchecking checkbox
+    Session.set_cell_attributes(socket.assigns.session_id, cell_id, %{value: "false"})
+
+    {:noreply, socket}
+  end
```

Comment:
I think we can add a hidden field alongside the checkbox to make sure it sends either true or false, that's what Phoenix helpers do. However this would affect a number of other places, so don't worry about this, I will refactor it later :)

Reply:
Sure thanks for the trick :+1: .

Reply:
Done!